python/tvm/relay/frontend/tflite.py

```diff
@@ -774,6 +775,21 @@ def convert_square(self, op):
 
         return out
 
+    def get_tensor_or_const_expr(self, tensor):
+        if self.has_expr(tensor.tensor_idx):
```

Comment:
This could do with a pydoc documentation for the interface.

Reply:
This is already addressed in previous commits.